README.md

```diff
@@ -27,7 +27,7 @@ This guide is structured around matching a Kubernetes offering around your exper
 
 ### I have no idea what I'm doing.
 
-That's okay! [Docker Desktop] and [MicroK8s] both are great places to start. Both of them handle the nitty-gritty of Kubernetes administration for you, and enable you to ease your way into experimentation.
+That's okay! [Docker Desktop] and [minikube] both are great places to start. Both of them handle the nitty-gritty of Kubernetes administration for you, and enable you to ease your way into experimentation.
```

Comment:
@aelindeman I'm assuming you meant for this, since the section has minikube, not MicroK8s

Reply:
yeah good catch, I swapped the sections for MicroK8s and minikube at some point